train.py

```
@@ -25,11 +25,11 @@
 from flame.parallelisms.parallelize_fla import parallelize_fla
 from flame.parallelisms.pipeline_fla import pipeline_fla
 from flame.utils import device_module, device_type
+from torchtitan.float8 import Float8Converter
 from torchtitan.logging import init_logger, logger
 from torchtitan.model_converter import build_model_converters
```

Ruff (0.8.2)

`torchtitan.model_converter.build_model_converters` imported but unused

Remove unused import: `torchtitan.model_converter.build_model_converters`

(F401)